en/docs/install-and-setup/perfromance-tuning-and-test-results/tuning-performance.md

```diff
@@ -384,28 +387,28 @@ validationQuery = "SELECT 2"
 </code></pre>
 </div>
 </div>
-<p>Note that you set the <code>               &lt;testOnBorrow&gt;              </code> element to <code>               true              </code> and provide a validation query (e.g., in Oracle, <code>               SELECT 1 FROM DUAL              </code> ), which is run to refresh any stale connections in the connection pool. Set a suitable value for the <code>               &lt;validationInterval&gt;              </code> element, which defaults to 30000 milliseconds. It determines the time period after which the next iteration of the validation query will be run on a particular connection. It avoids excess validations and ensures better performance.</p>
+<p>Note that you set the <code>               &lt;testOnBorrow&gt;              </code> element to <code>               true              </code> and provide a validation query (e.g., in Oracle, <code>               SELECT 1 FROM DUAL              </code> ), which is run to refresh any stale connections in the connection pool. Set a suitable value for the <code>               &lt;validationInterval&gt;              </code> element, which defaults to 30000 milliseconds. It determines the time period, after which the next iteration of the validation query will be run on a particular connection. It avoids excess validations and ensures better performance.</p>
 </div></td>
 </tr>
 </tbody>
 </table>
 
-##### Registry indexing configurations
+### Registry indexing configurations
 
-The registry indexing process is only required to be run on the API Publisher and Developer Portal nodes. To disable the indexing process from running on the other nodes (Gateways and Key Managers), you need to following in the `<API-M_HOME>/repository/conf/deployment.toml` file.
+The registry indexing process is only required to be run on the API Publisher and Developer Portal nodes. To disable the indexing process from running on the other nodes (Gateways and Key Managers), you need to add the following configuration section in the `<API-M_HOME>/repository/conf/deployment.toml` file.
```

Comment:
what does it index? when you say registry indexing , that is an internal detail.. but for a user it is different, for e.g. APIs.

Reply:
@chamilaadhi , please comment.

Reply:
actually this is not related to user. this configuration is done by system admin who maintains the server and registry indexing should be a familiar concept for sysadmin who maintains a wso2 server

Reply:
but why do they need to know what gets indexed? What value addition does it give?

Reply:
Updated in https://github.com/wso2/docs-apim/pull/798/commits/7554bc2eb4e58f59fa40bf8df0f43fe9e5b78b7b